src/main/java/com/synopsys/integration/alert/component/scheduling/SchedulingDescriptorActionApi.java

```diff
@@ -49,8 +47,8 @@ public SchedulingDescriptorActionApi(final TaskManager taskManager) {
 
     @Override
     public FieldModel readConfig(final FieldModel fieldModel) {
-        final Optional<Long> accumulatorNextRun = taskManager.getDifferenceToNextRun(BlackDuckAccumulator.TASK_NAME, TimeUnit.SECONDS);
-        fieldModel.putField(SchedulingDescriptor.KEY_ACCUMULATOR_NEXT_RUN, new FieldValueModel(List.of(accumulatorNextRun.map(String::valueOf).orElse("")), true));
+        final String accumulatorNextRun = taskManager.getDifferenceToNextRun(OnDemandTask.TASK_NAME, TimeUnit.SECONDS).map(String::valueOf).orElse("");
+        fieldModel.putField(SchedulingDescriptor.KEY_ACCUMULATOR_NEXT_RUN, new FieldValueModel(List.of(accumulatorNextRun), true));
```

Comment:
We may want to iterate through the providers and have keys inserted with a prefix and the provider name i.e. 
accumulator.next.run.blackduck, accumulator.next.run.polaris

Something like that.